src/image.rs

```diff
@@ -16,40 +16,53 @@ use crate::{ErrorKind, Renderer};
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct ImageId(DefaultKey);
 
-/// Image format: `Rgb8`, `Rgba8`, `Gray8`.
+/// Specifies the format of an image's pixels.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub enum PixelFormat {
+    /// 24-bit RGB image format (8 bits per channel)
     Rgb8,
+    /// 32-bit RGBA image format (8 bits per channel, including alpha)
     Rgba8,
+    /// 8-bit grayscale image format
     Gray8,
 }
 
 bitflags! {
-    /// Image flags (eg. repeat, flip, mipmaps, etc.)
+    /// Represents a set of flags that modify the behavior of an image.
     #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
     pub struct ImageFlags: u32 {
-        const GENERATE_MIPMAPS = 1;     // Generate mipmaps during creation of the image.
-        const REPEAT_X = 1 << 1;        // Repeat image in X direction.
-        const REPEAT_Y = 1 << 2;        // Repeat image in Y direction.
-        const FLIP_Y = 1 << 3;          // Flips (inverses) image in Y direction when rendered.
-        const PREMULTIPLIED = 1 << 4;   // Image data has premultiplied alpha.
-        const NEAREST = 1 << 5;         // Image interpolation is Nearest instead Linear
+        /// Generates mipmaps during the creation of the image.
+        const GENERATE_MIPMAPS = 1;
+        /// Repeats the image in the X direction when rendered.
```

Comment:
```suggestion
        /// Repeats the image in the X direction when rendered into an area wider than the image.
```

(same for the other)